scripts/ci/tags.yaml

```diff
@@ -97,7 +97,7 @@ posix:
   files:
     - zephyr/lib/posix/
 
-# cbprintf:
-#    files:
-#        - zephyr/lib/os/cbprintf*
-#        - zephyr/lib/posix
+wifi:
+  files:
+    - nrf/drivers/wifi
```

Comment:
`modules/hostap/` and `../modules/lib/hostap` too.

Reply:
Thanks, fixed.
The root folder is NCS, so it would be `modules/lib/hostap`

Reply:
And `modules/hostap` is actually at `sdk-nrf`